WSL/connect-usb.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Connect USB devices to WSL 2 
```

Comment:
```suggestion
title: Connect USB devices
```

Suggesting this edit so we can try and move away from "WSL 2" and the  norm can just become "WSL" :)